src/components/Dropdown/Dropdown.js

```diff
@@ -11,10 +13,9 @@ const Item = styled.div`
   overflow: hidden;
   text-overflow: ellipsis;
   white-space: nowrap;
-  padding-right: 28px;
-  padding-left: 12px;
+  padding-right: ${spacing('medium')};
```

Comment:
I'm thinking how `spacing('large')` looks better here to me than `medium`, especially when considering search filters:

`large`
![image](https://user-images.githubusercontent.com/1216874/49220561-d8c2dc00-f3d6-11e8-8592-4b6873540b7a.png)

`medium`
![image](https://user-images.githubusercontent.com/1216874/49220602-ebd5ac00-f3d6-11e8-81ec-0a958aa4c5e8.png)

Either is fine though, so let it be your call :)

Reply:
I thought that `medium` looked more symmetrical, but now I realise that the real issue with the spacing here is that the dropdown content can be so much larger (or smaller) than the width of this filter button. I made #434 to address this.